### PR TITLE
[#9509] Chat onboarding tooltip needs fixes to scroll behaviour

### LINKED ISSUE #9509: Chat onboarding tooltip needs fixes to scroll behaviour
# Problem

The current implementation of the chat onboarding tooltip has issues that relate to scrolling. 

- First, if the list of chats is empty, the screen can't be scrolled. This should be fixed to accommodate scrolling, when the list is empty.

- Secondly, after the tooltip is dismissed, the list still counts its height for the purpose of scrolling. This can be best observed on the video below

- Third, the tooltip doesn't have the bottom margin of 64, it should be possible to scroll so it's not covered by the + button.

[Link to video demonstrating issues](https://www.dropbox.com/s/7kn9gcnj26g0sx8/onboardingscroll.mp4?dl=0)

## Implementation

Implement the tooltip properly so it behaves just like a regular item on a scrollable list

## Acceptance Criteria

All of the above were fixed

## Notes

Thank you kindly!

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #9509

### CODE REVIEW

**Comment by errorists** [comment]
Tested on Android. Works GREAT now! 🙌  

Two minor corrections that have to do with assets used,

Screenshot 2019-11-26 at 14 52 01

the icon here isn't wide enough and appears clipped on right edge. [I exported the correct asset once again](https://www.dropbox.com/sh/0kfk5max8pdrsrt/AAA9f2W0ZkQF5L0UrEyc2aYya?dl=0), please try using it :) 

Second one, the close asset is using the tiny-Close icon, it's only 16 large and appears a bit small. instead it should be using the [close-Circle here](https://www.figma.com/file/cb4p8AxLtTF3q1L6JYDnKN15/Index?node-id=4094%3A1755) which is 24 in size. sorry for mislabeling them in the Figma!

**Comment by churik** [comment]
@flexsurfer one issue here.
In nightly after closing tooltip and relogin label "Your chat will appear here" was shown, but currently - empty screen.
That is the reason of failure  `test_home_view` in this PR.

Steps:
- create new multiaccount
- close chat onboarding tooltip
- relogin

Expected result: "Your chat will appear here"
2:04 in https://app.saucelabs.com/tests/7eb8e2050c7c44aa9903821899b5eff9?auth=1551eabda21aee36331ce583b3e7b61e#53

Actual result: empty screen
3:17 https://app.saucelabs.com/tests/b610437ba6754fa08fa023e4b63f73f8?auth=9722435600958d31923a0fada6484d54#72

**Comment by flexsurfer** [comment]
@churik @errorists thanks, fixed

**Review by errorists — APPROVED** [review]

File: src/status_im/ui/screens/home/views.cljs [hunk 1]
```diff
@@ -187,15 +184,12 @@
...
-            (if (and (not search-filter)
-                     (empty? all-home-items))
-              [home-tooltip-view true hide-home-tooltip?]
-              [home-items-view
-               search-filter
-               chats
-               all-home-items
-               hide-home-tooltip?
-               filter.views/search-input-state])])]
+            [home-items-view
```
Comment (vitvly) [inline src/status_im/ui/screens/home/views.cljs:187]: Not introduced here, but wondering why we pass 4 params that are currently not used.

**Review by vitvly — APPROVED** [review]

**Comment by churik** [comment]
@flexsurfer in commit `683d21d`  tooltip doesn't appear after creating account

**Comment by flexsurfer** [comment]
@churik thanks! fixed

**Pull request merged**